### DISCUSSION TITLE
账号密码错误，如何粗暴找回，目前登录不上去

### DISCUSSION THREAD

**Discussion by ZYyyh-cool** [halo-dev/halo#2165] [Q&A] [answered]
### 描述一下此特性

账号密码错误，如何粗暴找回，目前登录不上去

**Comment by ruibaby** [comment]
请正确选择 issue 模板，这个问题不属于新特性建议。

**Reply by ruibaby** [reply]
我将这个问题转移到了 discussion。

**Comment by ruibaby** [comment, marked as answer]
https://docs.halo.run/user-guide/faq#忘记了管理员密码如何重置
